modular_zubbers/code/game/machinery/computer/arcade/arcade_weights_oh_god.dm

```diff
@@ -0,0 +1,15 @@
+GLOBAL_LIST_INIT(arcade_prize_pool_oh_god, list(
+	/obj/item/card/emag/one_shot = 1,
+	/obj/item/clothing/mask/whistle/minigame = 1,
+	/obj/item/clothing/shoes/clown_shoes/banana_shoes = 1,
+	/obj/item/clothing/shoes/gunboots/disabler = 1,
+	/obj/item/clothing/suit/hooded/wintercoat/skyrat/syndicate = 1,
+	list(
+		/obj/item/clothing/under/syndicate/skyrat/tactical = 1,
+		/obj/item/clothing/under/syndicate/skyrat/tactical/skirt = 1,
+	) = 1,
+	/obj/item/greentext/quiet = 1,
+	/obj/item/grenade/chem_grenade/teargas/moustache = 1,
+	/obj/item/spellpacket/lightningbolt = 1,
+	/obj/item/storage/belt/champion/wrestling = 1
```

Comment:
The arcade machines already have a low chance to spawn a pulse rifle. We don't need everything to give out a chance for admin bus loot even if it's 1/5000. Specifically ``/obj/item/clothing/mask/whistle/minigame``, ``/obj/item/clothing/shoes/gunboots/disabler``, ``/obj/item/grenade/chem_grenade/teargas/moustache``, and ``/obj/item/storage/belt/champion/wrestling`` should not be things people can obtain.

Reply:
It's less than a 1 in 5000 chance. A 1 in 8751 chance. It's even lower than that when you consider the other items in the "oh god" category.

Reply:
I'll remove the minigame whistle, the moustache, and the wrestling belt, but the disabler boots stay on during sex.

Reply:
There are tons of 1/5000 items you can add that are nice treats but aren't immediately game-breaking. Like a bag of holding or an advanced jetpack module as two examples. Think of items that you can get with the best possible space-ruin loot. The biggest issue with these items is how powerful they are and how much they will incentivize extremely poor behavior. The disabler boots still fall under this category of highly annoying when realistically these should be items that are bragging rights but don't hold a high impact on the round at their worst possible use. Imagine if a security officer got the disabler boots, as an example. Hell, even Hilbert's hotel would be something that's cool. These are just some example items I can think on the top of my head